src/elements/ProjectSettingsGeneral/projectSettingsGeneral.scss

```diff
@@ -1,12 +1,22 @@
 @import '../../scss/colors';
+@import '../../scss/mixins';
 
 .settings {
-  &__artifact-path {
-    margin-bottom: 25px;
+  &__artifact-path,
+  &__goals,
+  &__description,
+  &__labels,
+  &__owner,
+  &__source {
+    margin-bottom: 15px;
+  }
 
+  &__artifact-path {
     &-link {
+      color: $topaz;
```

Comment:
according to stylelint rules the color property should be after display:block and margin-top, and font-weight should goes before font-size

Reply:
```
display: block;
      margin-top: 5px;
      font-weight: 400;
      color: $topaz;
      font-size: 0.75rem;
```